src/pages/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import useAxios from "../../hooks/useAxios";
+import Carousel from "../../components/Carousel/Carousel";
+import LoadingSpinner from "../../components/LoadingSpinner/LoadingSpinner";
+import classes from "./LandingPage.module.scss";
+import {
+  LANDING_PAGE_LOADING_TEXT,
+  LANDING_PAGE_PARAGRAPH,
+  LANDING_PAGE_START_APPLICATION_BUTTON,
+} from "../../static";
+
+/**
+ * Landing/Home page is the first page that loads on the user's screen when user lauches the app,
+ *
+ * @returns {JSX} a slider type carousel and a section where user can read about, click and start the application process
+ */
+
+const LandingPage = () => {
+  const [data, setData] = useState([]);
+  const {
+    error,
+    isLoading,
+    fetchData: getImageOfTheDay,
+  } = useAxios(
+    `https://api.nasa.gov/planetary/apod?count=16&api_key=${process.env.REACT_APP_API_KEY}`
```

Comment:
count should be in variable, and all logic to go around it.
 `https://api.nasa.gov/planetary/apod?count={COUNT}&api_key=${process.env.REACT_APP_API_KEY}`
 So when someone change that input variable, whole Carousel component should work properly